### DISCUSSION TITLE
[Feature Request] Global response handler

### DISCUSSION THREAD

**Discussion by Elfayer** [Akryum/vue-apollo#485] [Q&A]
There is no such thing as a "global response handler" in the doc API: https://akryum.github.io/vue-apollo/api/apollo-provider.html#constructor

### Problem
The global error handler will only get triggered on schema/network errors. If the API `throw`s some error, it'll still return a 200 OK containing `errors: []`, I'd like to notify errors globally, but I need a global response handler for that.
 For example, axios provides both (https://github.com/axios/axios#interceptors).

**Comment by Akryum** [comment]
You can use Apollo Links, such as the ErrorLink.
